src/services/roles.ts

```diff
@@ -1,4 +1,4 @@
-import { logEvent } from "../functions/authorizer";
+import { ILogEvent } from "../models/ILogEvent";
 
 export type Access = "read" | "write" | "view";
 
```

Comment:
Im sure this function can just be improved but looks out of scope of the changes youve made.